Dockerfile

```diff
@@ -32,6 +32,7 @@ COPY webpack webpack
 ENV NODE_ENV "production"
 ARG CI_COMMIT_SHA
 ENV CI_COMMIT_SHA ${CI_COMMIT_SHA}
+RUN echo "{ \"version\": \"${CI_COMMIT_SHA}\", \"date\":\"$(date)\" }" > app/version.json
```

Comment:
this is not necessary. cf. https://github.com/elifesciences/elife-xpub/blob/develop/webpack/webpack.production.config.js#L41-L43

Reply:
This looks to be config and not baked in on build

Reply:
Thanks for pointing this out, but we do need this information as it is required for the footer. This is there so that people within eLife who test the product can have something to reference when things go wrong.